tools/l10n/initialize.sh

```diff
@@ -0,0 +1,4 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Is this file used anywhere?

Reply:
Ops. Initially I wanted to put the repository initialization here. But I moved it into the other scripts so that they work on their own. I'll remove this from the patch.